healthcareai/common/transformers.py

```diff
@@ -54,7 +56,17 @@ def transform(self, X, y=None):
         # Return if not imputing
         if self.impute is False:
             return X
-
+        
+        #Warn users if %nan is too high
+        for c in X:
+            pct_impute = X[c].isnull().sum() / len(X)
+            if pct_impute > self.max_impute:
+                warnings.warn("{0:.2f}% of data for column '{1}' is missing. Imputed "
+                              "values may be invalid.".format(pct_impute*100, c),
+                               RuntimeWarning)
+        
+        #Alternative fill, only fill maximum number of values based on max_impute
+        #result = X.fillna(self.fill, limit=len(X)//(1/self.max_impute))
```

Comment:
I'm not sure how this will affect downstream algorithms. Do you have any sense for this? If not, I'd say create another issue and remove this commented code from this PR.